### ISSUE TITLE
Grayed char reset dosent last

### ISSUE THREAD

**Issue by eclipsaa** [BarleyFlour/WrathVisualAdjustmentsMod#19]
My hole party is graying out every time a new area loads.   The reload char but fixs it but i have to do it in EVERY area.  Plus i have to go back in to the char sheet and check the show class armor button because it also breaks that option.  

Only thing i have done is use a custom hair color on my main char and changed my main chars class armor to slayer

**Comment by Tahtivaeltaja** [comment]
Likewise. Been experiencing the same issuesfor the last couple of hours and than remembered I changed the default outfit of one of the characters. Returned everything to default and the characters are not "grayed out" any more. Their appearance remains normal on zone changes as well, as long as I don't change their class outfit..
I've been checking out the Owlcat forums for any mention of this as well, found nothing.  So it has to be connected to this mod.

**Comment by eclipsaa** [comment]
I can change the class outfit and stuff its fine.  its just the customer colors thats messing everything up.  Witch is really sad because the base game color options are all really bad.   I really wanted the custom colors for my hair and the 2 outfit colors.

**Comment by cnieuweboer** [comment]
I get the grey skin (and blonde hair) issue too on any area transition with a loading bar. When the new area loads the colors are fine, but after half a second the skin and hair colors bug out.

After fully removing/installing the mod, not changing any settings but just having it active I don't get the grey skin issue. But when changing setting in the mod, even if it's just clicking the "default" button in "select outfit" the issue returns. The grey skin also appears often when changing any of the "select doll" and select override" options. When waiting a few moments, and going back and forth once or twice usually fixes that. 

This is always fixed temporary by clicking  the "Fix Grey Characters" button.

Save file to easily reproduce (use the manage cruisade button and exit): [https://www.dropbox.com/s/mwpvgqsnx9i5q22/Quick_2.zip?dl=0](url)

**Comment by typoking7** [comment]
I've got the same problem in 1.09.0

**Comment by Ebonmark** [comment]
Not sure if it's feasible, but if there was a checkbox that could be ticked off to have the mod automatically 'Fix Grey Characters' on zone load in, that would be handy.

**Comment by eclipsaa** [comment]
Yeah that would help i think.   Run on new area load or something.   Though im noticing that this might not be a full mod issues but something in game as well.  If you play around with the hiding class armor in the char sheet your char colors get all messed up sometimes.  I wonder if these issues are connected?

**Comment by Ebonmark** [comment]
Yes I believe the issue is producible even without VisualAdjustments mod, it just exacerbates the issue. I believe it would be Owlcat's responsibility to fix the root cause but until then, an auto 'fix grey character' setting would go a long way.

**Comment by nbelley** [comment]
So, what's the deal, is it owlcat's bug or this mod? I mean, its fun having a mod to change character appearances but having to rebuild characters on each load is not fun nor immersive at all...
